Tables docs improvements (#33059)
Azure/azure-sdk-for-net: sdk/tables/Azure.Data.Tables/README.md

```

If you prefer LINQ style query expressions, we can query the table using that syntax as well.
To demonstrate this syntax, you'll need a strongly typed model such as the one below:

```C# Snippet:TablesSample2DefineStronglyTypedEntity
// Define a strongly typed entity by implementing the ITableEntity interface.
public class OfficeSupplyEntity : ITableEntity
{
    public string Product { get; set; }
    public double Price { get; set; }
    public int Quantity { get; set; }
    public string PartitionKey { get; set; }
    public string RowKey { get; set; }
    public DateTimeOffset? Timestamp { get; set; }
    public ETag ETag { get; set; }
}
```

> Given this model class definition, here is how you'd write a query.

Reviewer: ```suggestion
Given this model class definition, here is how you'd write a query:
```

## Troubleshooting

> When you interact with the Azure table library using the .NET SDK, errors returned by the service correspond to the same HTTP

Reviewer: ```suggestion
When using the Azure tables library, errors returned by the service are reported using the same HTTP
```

Reviewer: This read awkwardly to me.   I'm not sure that I'm in love with my suggestion either, but the "Azure tables library" and ".NET SDK" distinction was where I struggled a bit and had to read a few times to understand.
